lib/presentation/model/chat_list/chat_selection_actions.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:equatable/equatable.dart';
+
+enum SelectionType {
```

Comment:
i think no need, slection or not is just a boolean

Reply:
Vote use enum